x-pack/plugins/task_manager/server/integration_tests/managed_configuration.test.ts

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
So, new directory `integration_tests`?  I guess the idea with these is that they actually launch a task manager to  operate on, so a little different than our other jest tests.  Cool - I could see us adding more tests here!

Reply:
Exactly! There's a few plugins that use the concept of [jest integration tests](https://github.com/elastic/kibana/blob/master/x-pack/scripts/jest_integration.js) to have something higher level than a unit test yet lower level than an API integration test to make sure it all works together. I agree there's a lot of potential here for future tests.

I realized the test ran by the `node scripts/jest` script instead of the `node scripts/jest_integration`. I'll look into it.